HousingRepairsOnlineApi/Helpers/RepairTypeHelper.cs

```diff
@@ -0,0 +1,10 @@
+﻿
```

Comment:
This duplicates the already existing [RepairType](https://github.com/Newarkandsherwood/housing-repairs-online-api/blob/main/HousingRepairsOnlineApi/Helpers/RepairType.cs) class.

Is it possible to reuse the existing one to remove the duplication?

The only usage of this class is [here](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/30/files#diff-2d7898aa4337b404555629732503b7d0b37f54f3384c9b388fb308e30d7a4fbdR43), which passes down the value through multiple layers (use case, gateway, helper, etc) and seems like it _could_ (see next paragraph) be replaced with the existing `RepairType`.

The difference between the values in the classes is letter casing (all upper case vs Pascal case)
The actual value is used in the database query ([here](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/30/files#diff-96659e09d0811df42f48f09a6c64e33a59085fd88914d0fcf123af899b83dd9cR19)) which likely should be a case insensitive check.